SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialEventTracerUserInterface.h

```diff
@@ -81,6 +71,24 @@ class SPATIALGDK_API USpatialEventTracerUserInterface : public UBlueprintFunctio
 	UFUNCTION(BlueprintCallable, Category = "SpatialOS|EventTracing", meta = (WorldContext = "WorldContextObject"))
 	static bool GetActiveSpanId(UObject* WorldContextObject, FUserSpanId& OutUserSpanId);
 
+	// ----- C++ Specific API -----
+
+	/**
+	 * EXPERIMENTAL
+	 * Will trace an event using the input data and associate it with the input SpanId
+	 * (This API is subject to change)
+	 */
+	static FUserSpanId TraceEvent(UObject* WorldContextObject, const FString& EventType, const FString& EventMessage,
```

Comment:
Think there is a chance here to also optimize this (no alloc / no virtual call) but don't know if it's necessary

Reply:
Probably won't be the bulk of the events